styles.css

```diff
@@ -31,17 +31,50 @@ body {
     background-color: #e5e5e5;
   }
 
-  .menu {
+  .sections {
```

Comment:
- [ ] I can see a few differences between your design and Figma design,  As mentioned by the previous reviewer it is very important to stick to the Figma design as much as possible in order to have a beautiful website and good user experience 👍. As we can see, your Xmark design and menu option font size and weight, spacing(padding/margin), and position are a bit different, please check the screenshots below ⬇️ to see the differences.  Please make sure that your mobile menu design is similar to the figma template  👍


####  Your design
![mobbbbbb](https://user-images.githubusercontent.com/6615394/223575889-3973aba0-0c76-49f2-9ca3-3f1a3a5e863d.jpg)


#### Figma design
![223529223-03b4d05c-fd1c-4b95-821c-ce6be91a4aa6](https://user-images.githubusercontent.com/6615394/223576274-0a85c67c-d2c6-4248-b7fd-d2b7357117e0.png)

Reply:
Thank you for your time @codecaiine. This is my last review so I wanted to be sure of what I need to change.
If I understand you well, I need to work on the following points:
- Move the button a little bit to the left;
- Increase the space between the menu items;
- Reduce the font size of the menu items;
Thanks, and please confirm If I'm right.
Best,